test/conformance/ptp/ptp.go

```diff
@@ -279,6 +281,32 @@ var _ = Describe("[ptp]", func() {
 					waitUntilLogIsDetected(testPtpPod, 3*time.Minute, "Profile Name: test-slave")
 				})
 			})
+
+			It("Should retrieve the details of hardwares for the Ptp", func() {
+				By("Getting the version of the OCP cluster")
+
+				ocpVersion, err := getOCPVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ocpVersion).ShouldNot(BeEmpty())
+
+				By("Getting the version of the PTP operator")
+
+				ptpOperatorVersion, err := getPtpOperatorVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ptpOperatorVersion).ShouldNot(BeEmpty())
+
+				By("Getting the NIC details of the PTP enabled interfaces")
+
+				for _, pod := range ptpRunningPods {
+					ptpDiscoveredInterfaces := ptpDiscoveredInterfaceList(NodePtpDeviceAPIPath + pod.Spec.NodeName)
+					fmt.Println(ptpDiscoveredInterfaces)
+
+					var mapping = getNICInfo(pod)
+
+					fmt.Println(mapping)
+					Expect(mapping).ShouldNot(BeEmpty())
+				}
+			})
```

Comment:
ptpconfigs are now stored in the testconfig singleton, maybe we can just dump the test config object. ptp configs are at:
https://github.com/openshift/ptp-operator/blob/24000706580808de3f7046d272edc754344acc67/test/utils/testconfig/testconfig.go#L76-L77

Reply:
is the goal of the jira to test the ptpconfig or dump them ?

Reply:
Thanks, I would make changes to dump ptpConfig.